packages/client/src/object/convertWireToOsdkObjects/createOsdkObject.ts

```diff
@@ -73,6 +73,65 @@ const objectPrototypeCache = createClientCache(
   },
 );
 
+if (process.env.NODE_ENV !== "production") {
+  const installed = Symbol();
+  const gw: any = typeof window === "undefined" ? global : window;
+  if (!(installed in gw)) {
+    gw[installed] = true;
+
+    gw.devtoolsFormatters ??= [];
+    gw.devtoolsFormatters.push({
+      header: function(object: any) {
+        const raw = object[RawObject];
+        if (raw == null) return null;
+
+        return [
+          "div",
+          {},
+
+          `Osdk.Instance<${raw.$apiName}> { $primaryKey:`,
+          ["object", { object: raw.$primaryKey }],
+          `, $title:`,
+          ["object", { object: raw.$title }],
+          `, ... }`,
+        ];
+      },
+      hasBody: function(object: any) {
+        return object[RawObject] != null;
+      },
+      body: function(object: any) {
+        const raw = object[RawObject];
+        if (raw == null) return null;
+
+        return [
+          "ol",
+          {
+            style: `
+            list-style-type: none;
+            padding-left: 0;
+            margin-top: 0;
+            margin-left: 18px
+          `,
+          },
+          ["li", {}, "$raw:", ["object", { object: raw }]],
+          // ...Object.keys(raw).map(key => {
+          //   return [
+          //     "li",
+          //     {},
+          //     [
+          //       "span",
+          //       { style: "color: #888888" },
+          //       `${key}: `,
+          //     ],
+          //     ["object", { object: raw[key] }], // ["span", {}, `${key}: `]],
```

Comment:
Did you mean to keep these?

Reply:
I was on the fence about which style to use so i left it so it would be easy for us to change our mind/experiment